packages/quantic/force-app/main/default/lwc/quanticNumericFacet/quanticNumericFacet.js

```diff
@@ -11,46 +11,101 @@ import clearFilter_plural from '@salesforce/label/c.quantic_ClearFilter_plural';
 import collapseFacet from '@salesforce/label/c.quantic_CollapseFacet';
 import expandFacet from '@salesforce/label/c.quantic_ExpandFacet';
 
+/** @typedef {import("coveo").NumericFacetState} NumericFacetState */
+/** @typedef {import("coveo").NumericFacet} NumericFacet */
+/** @typedef {import("coveo").SearchEngine} SearchEngine */
+/** @typedef {import("coveo").NumericFacetValue} NumericFacetValue */
+
+/**
+ * The `QuanticNumericFacet` component displays facet values as numeric ranges.
+ * @example
+ * <c-quantic-numeric-facet engine-id={engineId} facet-id="myfacet" field="ytlikecount" label="Youtube Likes" numberOfValues="5" sort-criteria="descending" range-algorithm="even" formatting-function={myFormattingFunction} is-collapsed></c-quantic-numeric-facet>
+ */
 export default class QuanticNumericFacet extends LightningElement {
-  /** @type {import("coveo").NumericFacetState} */
-  // @ts-ignore TODO: Check CategoryFacetState typing and integration with LWC/Quantic
-  @track state = {
-    values: [],
-  };
-
-  /** @type {string} */
+  /**
+   * The ID of the engine instance the component registers to.
+   * @api
+   * @type {string}
+   */
+  @api engineId;
+  /** 
+   * A unique identifier for the facet.
+   * @api
+   * @type {string}
+   * @defaultValue Defaults to the `field` value.
+   */
   @api facetId;
-  /** @type {string} */
+  /**
+   * Specifies the index field whose values the facet should use.
+   * @api
+   * @type {string}
+   */
   @api field;
-  /** @type {string} */
+  /**
+   * The non-localized label for the facet. This label is displayed in the facet header.
+   * @api
+   * @type {string}
+   */
   @api label = 'no-label';
-  /** @type {string} */
-  @api engineId;
-  /** @type {number} */
+  /**
+   * The number of values to request for this facet, when there are no manual ranges.
```

Comment:
The mention of manual ranges is bugging me a little since it does not seem to be supported by the facet.

Reply:
@nathanlb I just saw [your comment](https://github.com/coveo/ui-kit/pull/1255#discussion_r720274921). :+1: